src/ast/pass_manager.cpp

```diff
@@ -27,7 +27,7 @@ void PassManager::AddPass(Pass p)
 PassResult PassManager::Run(std::unique_ptr<Node> node, PassContext &ctx)
 {
   Node *root = node.release();
-  if (bt_debug != DebugLevel::kNone)
+  if (bt_debug.find(DebugStage::Ast) != bt_debug.end())
```

Comment:
can we use `contains` yet?

Reply:
We're still at C++17. I left a comment in #2852 to transform these into `contains` when we move to C++20.